instaloader/instaloader.py

```diff
@@ -745,7 +745,8 @@ def download_stories(self,
                 self.context.log("[%3i/%3i] " % (count, totalcount), end="", flush=True)
                 count += 1
                 with self.context.error_catcher('Download story from user {}'.format(name)):
-                    downloaded = self.download_storyitem(item, filename_target if filename_target else name)
+                    path_target: Path = Path(filename_target.format(name))
```

Comment:
[Help needed]
Error: `instaloader/instaloader.py:748: error: Item "None" of "Optional[str]" has no attribute "format"`

Isn't it replaced with ":stories" when None is passed?